src/api/app/assets/stylesheets/webui2/application/package.scss

```diff
@@ -0,0 +1,7 @@
+.fuzzy-time {
+  text-decoration: underline dashed #069;
```

Comment:
Just wrap this in `u`
```
<p><u>This line of text will render as underlined</u></p>
```

Reply:
We are not trying to recreate the old UI. We are trying to do what makes sense :)